server/vcap-services.es6

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+let _ = require('lodash');
```

Comment:
If this is the only place you are using `lodash` you can eliminate this dependency by using native `es6` array.Prototype.find(). https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find